### PR TITLE
Divyanshu | Test | Connect bank account button is coming on dashboard in Plaid non supported countries

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cukht8h

### CODE REVIEW

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 1]
```diff
@@ -379,7 +379,7 @@ <h3 class="section-head mb-2">{{ localeData?.ecommerce?.shopify_form_heading }}<
                 </div>
             </mat-tab>
 
-            <mat-tab [label]="localeData?.payment?.tab_heading" *ngIf="!isIciciBankSupportedCountry || iciciAllowedCompanies.includes(activeCompany?.uniqueName)">
+            <mat-tab [label]="localeData?.payment?.tab_heading" *ngIf="(!isIciciBankSupportedCountry || iciciAllowedCompanies.includes(activeCompany?.uniqueName)) && isPlaidSupportedCountry">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html]:
```suggestion
            <mat-tab [label]="localeData?.payment?.tab_heading" *ngIf="(!isIciciBankSupportedCountry || iciciAllowedCompanies.includes(activeCompany?.uniqueName)) || isPlaidSupportedCountry">
```

Comment (ravinderwalkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html]: Line 424 pe bhi condition hona chahiye - isPlaidSupportedCountry

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html]: Done

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 2]
```diff
@@ -379,7 +379,7 @@ <h3 class="section-head mb-2">{{ localeData?.ecommerce?.shopify_form_heading }}<
                 </div>
             </mat-tab>
 
-            <mat-tab [label]="localeData?.payment?.tab_heading" *ngIf="!isIciciBankSupportedCountry || iciciAllowedCompanies.includes(activeCompany?.uniqueName)">
+            <mat-tab [label]="localeData?.payment?.tab_heading" *ngIf="(!isIciciBankSupportedCountry || iciciAllowedCompanies.includes(activeCompany?.uniqueName)) || isPlaidSupportedCountry">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html]:
```suggestion
            <mat-tab [label]="localeData?.payment?.tab_heading" *ngIf="iciciAllowedCompanies.includes(activeCompany?.uniqueName) || isPlaidSupportedCountry">
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**